Source/storm/storm.cpp

```diff
@@ -215,6 +215,41 @@ void setIniFloat(const char *keyname, const char *valuename, float value)
 	getIni().SetDoubleValue(keyname, valuename, value);
 }
 
+/**
+ * @brief Check item exists in comma-sparated list
+ */
+bool IniFound(const char *list, const char *item)
+{
+	const char *look = strcasestr(list, item);
+	if (!look)
+		return false;
+
+	look += strlen(item);
+	while (*look == ' ')
+		look++;
+	return (bool) strchr(",;:", *look); // delimters including '\0'
+}
+
+/**
+ * @brief Find needle pattern in haystack, ignoring case
+ */
+const char *strcasestr(const char *haystack, const char *needle)
+{
+	const char *look, *pp, *qq;
+	assert(haystack && needle);
+
+	for ( look = haystack; *look; look++ ) {
+		if (tolower(*look) == tolower(*needle)) {
+			pp = needle + 1; qq = look + 1;
+			while (*pp && (tolower(*pp) == tolower(*qq)))
+				pp++, qq++;
+			if (!*pp)
+				return look;	// match found!
+		}

```

Comment:
I cannot believe there is no such method already in stdlib. Is there really not?